src/openrct2/rct1/S4Importer.cpp

```diff
@@ -1603,7 +1603,7 @@ namespace RCT1
                     dst2->SetHasQueueBanner(src2->HasQueueBanner());
                     dst2->SetEdges(src2->GetEdges());
                     dst2->SetCorners(src2->GetCorners());
-                    dst2->SetAddition(src2->GetAddition());
+                    dst2->SetAddition(0);
```

Comment:
Is this intended?

Reply:
It is, yes. It gets properly converted and set later on, so this avoids potentially writing garbage to the field.